clients/admin-ui/src/features/common/ConfirmationModal.tsx

```diff
@@ -66,7 +68,7 @@ const ConfirmationModal = ({
           <Button
             variant="outline"
             mr={3}
-            onClick={onClose}
+            onClick={onCancel || onClose}
```

Comment:
is there ever a case where we want to cancel but _not_ close the modal? if not, I feel like this might be better as

```js
onClick={() => {
  if (onCancel) {
    onCancel();
  }
  onClose();
}}
```

so that components don't have to worry about adding cancel logic _and_ closing logic

Reply:
I see what you mean. I did this because I needed a separate function from `onClose`. The reason for that is the modal needs an `onClose` for when a user clicks on the blurred overlay that surrounds that modal. I needed to add a custom cancel function because now the cancel will handle doing the other "non upgrade" option in the modal.


I like your proposed solution. I'll update it to do it that way :+1: